### ISSUE TITLE
Boxed: preserve identity when GType is none

### ISSUE THREAD

**Issue by romgrk** [romgrk/node-gtk#40]

**Comment by Place1** [comment]
Yeah I noticed this when testing signals, finding some callback arguments
had the wrong type. I’d be keen to look into what happens in gjs in this
situation.
